new_front_page_mockup.md

```diff
@@ -0,0 +1,73 @@
+{logo}
+
+// 25 word message describing Operate First, something like:
+
+**Operate First is a concept to incorporate operational experience into software projects.**
+
+[CTAs]
+How it works →
+Bring your code →
+Read more → “How to open source cloud operations” by Marcel Hild
+
+[fold]-----------------------------------------------------------------------------------------------------------------
+
+// some kind of graphic to explain:
+
+Operate First Community of Practice: the community that practices operating first, sets the guidance for the method, and continually improves upon that method
+
+An Operate First Community Production Cloud: an environment in which you can operate first
+
+**Vision or motivation**
+
+// description of Operate First motivation and its impacts on developers, operators, and final product
+
+While open source has made software widely available, it has also exposed another barrier to entry: the ability to operate software in production. Proprietary services for operations undermine the open source model, so we must now shift our focus to open sourcing operations.
```

Comment:
Recommend you put a line break after each sentence (with two spaces still used between paragraphs.)

This makes it easier to do reviews because changes are not lost in a paragraph, but instead visible line by line.

Here is what the change looks like:

`While open source has made software widely available, it has also exposed another barrier to entry: the ability to operate software in production.
Proprietary services for operations undermine the open source model, so we must now shift our focus to open sourcing operations.`

That puts each sentence on its own line, but they still render as a paragraph (this works in MarkDown and AsciiDoc.)

Then if I were to suggest a change in one of the sentences, just one of the two lines would be marked as changed, and the difference between then is easier to see.

Reply:
@quaid I tried implementing this change, but they're now rendering as two separate lines for me - not sure why

Reply:
update: fixed it!